app/forms/candidate_interface/other_qualification_wizard.rb

```diff
@@ -0,0 +1,226 @@
+module CandidateInterface
+  class OtherQualificationWizard
```

Comment:
I can't see whether this new class is referenced from at all? 🤔

Reply:
Sorry - this was the result of a bad rebase. Steve had done some refactoring in another PR. Fixed now.